### PR TITLE
Fix release pipeline to use local nuget packages for testing

### PR DESCRIPTION
**Pull request by abhipsaMisra**

### CODE REVIEW

File: vsts/gatedBuild.ps1 [hunk 1]
```diff
@@ -18,7 +18,7 @@ docker ps -a
 #Load functions used to check what, if any, e2e tests should be run
 . .\vsts\determine_tests_to_run.ps1
 
-$runTestCmd = ".\build.ps1 -clean -build -configuration DEBUG -framework $env:FRAMEWORK"
+$runTestCmd = ".\build.ps1 -clean -build -configuration DEBUG -framework $env:FRAMEWORK -nobuildBeforeTesting"
```
Comment (timtay-microsoft) [inline vsts/gatedBuild.ps1:21]: Does casing matter on these flags? "-nobuild..." vs "-noBuild..."?

Comment (abhipsaMisra) [reply vsts/gatedBuild.ps1:21]: Ah, I would assume that it is. I'll fix this.

**Comment by abhipsaMisra** [comment]
Will be fixed in #1459

**Pull request closed without merging**